package.json

```diff
@@ -46,6 +45,7 @@
     "react-helmet": "^5.2.1",
     "react-intercom": "^1.0.15",
     "react-markdown": "^4.2.2",
+    "react-metomic": "^0.1.0",
```

Comment:
Hi all, Joel from Metomic here. If you bump this to 0.1.2 this should work now. Thanks for putting up with us!